packages/deployment/main.js

```diff
@@ -962,39 +981,38 @@ ${units}`;
       out.write(byGroup.all);
       out.write('  children:\n');
       for (const group of Object.keys(byGroup).sort()) {
-        if (group === 'all') {
-          continue;
+        if (group !== 'all') {
+          out.write(indent(byGroup[group], 4));
         }
-        out.write(indent(byGroup[group], 4));
       }
       break;
     }
 
     case 'play': {
       const [pb, ...pbargs] = args.slice(1);
       if (!pb) {
-        throw `Need: [playbook name]`;
+        throw Error(`Need: [playbook name]`);
       }
       if (!pb.match(/^\w[-\w]*$/)) {
-        throw `[playbook] ${JSON.stringify(pb)} must be a word`;
+        throw Error(`[playbook] ${JSON.stringify(pb)} must be a word`);
       }
       await inited();
-      return await doRun(playbook(pb, ...pbargs));
+      return doRun(playbook(pb, ...pbargs));
     }
 
     case 'run': {
-      const [host, ...cmd] = args.slice(1);
-      if (!host || cmd.length === 0) {
-        throw `Need: [host] [cmd...]`;
+      const [host, ...subcmd] = args.slice(1);
+      if (!host || subcmd.length === 0) {
+        throw Error(`Need: [host] [cmd...]`);
       }
       await inited();
       let runArg;
-      if (cmd.length === 1) {
-        if (cmd[0].match(shellMetaRegexp)) {
+      if (subcmd.length === 1) {
+        if (subcmd[0].match(shellMetaRegexp)) {
           // Already contains metacharacters.
-          runArg = `sh -c ${shellEscape(cmd[0])}`;
+          runArg = `sh -c ${shellEscape(subcmd[0])}`;
         } else {
-          runArg = cmd[0];
+          [runArg] = subcmd;
```

Comment:
This kind of thing inclines me towards thinking our default lint rc should remove the `prefer-destructuring` rule.

Reply:
The es-lint rules should agree across all our code unless there's a very good reason for them to differ. We should consider changes to our rules to apply across our entire code base, to be applied automatically and, ideally, atomically. Killing `prefer-destructuring` should be on the table then, as should `"trailing-comma": "all"`.

But we should not consider making any such changes only for one package.